dataset_generation/hi_wikisplit/Scraper.py

```diff
@@ -0,0 +1,174 @@
+import logging
```

Comment:
the wikisplit strategy could be run for any language. so, can we please rename this parent folder to just wikisplit?

Reply:
yes, that makes sense. I'll do that.